packages/main/src/components/protected/sidebar/components/Sidebar.jsx

```diff
@@ -51,7 +51,7 @@ const Sidebar = props => {
         document.querySelector("body").style.cursor = "col-resize";
```

Comment:
Make use of refs if you want to make changes to the dom.

Reply:
okay, we didn't make these changes

Reply:
ok @billmal071  we didn't make those changes